edsby.py

```diff
@@ -15,20 +19,46 @@
     This software is unofficial and not supported by CoreFour in any way.
 """
 
+
 class Edsby(object):
     def __init__(self, **kwargs):
-        self.edsbyHost = kwargs['host']
+        # Setup config
+        config = configparser.ConfigParser()
```

Comment:
I'm afraid I'm not a huge fan of including all of this code in the constructor, as it adds a great deal of complexity  (over 20 new lines!). In my opinion, the entire configuration loading/parsing process would be much better off if it were moved somewhere else.

Generally speaking, the goal is to keep PyEdsby as lean as possible. As an Edsby API wrapper, It should only contain methods that are directly related to sending/retrieving data from Edsby, and methods for parsing and processing these data into a useful format (where appropriate). 

Everything else (reading configuration files, generating reports, etc) would be better left to the developer using PyEdsby in their project, because that's the most flexible philosophy. Configuration management is something that should be performed in your program's main file, rather than in PyEdsby.

Just to give a hypothetical about focus, consider a situation where it's useful to generate a CSV file of your class rosters. This would be something that would be better implemented outside of PyEdsby. Doing so reduces the complexity of the Edsby class, which:
   - makes it easier to maintain
   - makes it easier to extend
   - eliminates unnecessary dependencies (don't need to add a CSV lib, import sys, or work with the filesystem
   - Simplifies PyEdsby development, as development focuses on its relevant components

I've been considering creating a second class which extends the main Edsby class (or perhaps even a subclass), and incorporates a number of tools (like a config reader) that build on PyEdsby's methods. Perhaps call it EdsbyTools? That would be the best place to implement something that returns Edsby instances based on a configuration file.